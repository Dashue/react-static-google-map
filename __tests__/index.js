// Add tests, don't be lazy.
