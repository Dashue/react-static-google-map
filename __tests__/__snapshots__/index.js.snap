// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components It renders an image with default props 1`] = `
<img
  src="https://maps.googleapis.com/maps/api/staticmap?size=test&scale=1&format=png&maptype=roadmap&markers=size:normal%7Clocation"
/>
`;
